src/Microsoft.AspNet.TestHost/TestServer.cs

```diff
@@ -11,6 +11,7 @@
 using Microsoft.AspNet.Http.Features;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
+using Context = Microsoft.AspNet.Hosting.Internal.HostingApplication.Context;
```

Comment:
This is awkward since hard-coding the generic type seems to defeat the purpose of generics int he first place but this is the cleanest way of implementing the TestServer.